Sources/ClientRuntime/Networking/Http/SdkHttpRequest.swift

```diff
@@ -29,11 +29,10 @@ public class SdkHttpRequest {
 
 extension SdkHttpRequest {
 
-    public func toHttpRequest(escaping: Bool = false) throws -> HTTPRequest {
```

Comment:
Escaping of paths is no longer performed anywhere but at the point the path is created.  As a result, SdkHttpRequest no longer has an option to encode or not.